storage/app/api/v1/attachments/controller.py

```diff
@@ -34,8 +29,8 @@ def upload_attachment_raw(data):
     raw_data_size = raw_data.getbuffer().nbytes
     uuid4 = str(uuid.uuid4())
     try:
-        s3_upload_raw_file(ATTACHMENTS_BUCKET_NAME,
-                           uuid4, raw_data, raw_data_size)
+        return s3_upload_raw_file(ATTACHMENTS_BUCKET_NAME,
+                                  uuid4, raw_data, raw_data_size)
```

Comment:
This chunk needs to be reverted - and add a check to make sure that the s3 response succeeded